src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -25,20 +25,39 @@ public class HostAssignmentContext
         public MSIContext MSIContext { get; set; }
 
         public string ZipUrl
+        {
```

Comment:
Can you return (string, string) from ZipUrl and keep the current logic instead of new property